mypy/constraints.py

```diff
@@ -126,15 +126,6 @@ def infer_constraints_for_callable(
     param_spec_arg_names = []
     param_spec_arg_kinds = []
 
-    incomplete_star_mapping = False
-    for i, actuals in enumerate(formal_to_actual):
```

Comment:
Looking at the code this should be `enumerate(arg_types)`. I am actually surprised that all tests still pass. Or maybe I simply forgot to write a test for this case.

Reply:
I will keep this as-is with a `type: ignore` and a `TODO` comment.